modules/ROOT/pages/policy-mule3-add-remove-headers.adoc

```diff
@@ -118,16 +118,16 @@ Perform the steps in this procedure to apply either of the following policies:
 
```

Comment:
change to the new policy name "Header Removal Policy"

Reply:
Done! Thanks!